using-mobile-extensions/mobile-core/mobile-core-release-notes.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## February 4, 2020
+
+**Android Identity 1.2.0** 
+
+- Added support for the optional `AdobeCallbackWithError` available in Android Core 1.5.0 on the following APIs: `appendVisitorInfoForURL`, `getUrlVariables`, `getIdentifiers`, `getExperienceCloudId`. When the AdobeCallbackWithError is used, a default 500ms timeout is enforced when retrieving the Mobile SDK values; an AdobeError is returned if the operation times out or is not successful.
```

Comment:
Please restructure as follows:
"* Added support for the optional `AdobeCallbackWithError` API that is available in Android Core version 1.5.0 on the following APIs: 
          * `appendVisitorInfoForURL`
          * `getUrlVariables`
          * `getIdentifiers`
          * `getExperienceCloudId`

   When the `AdobeCallbackWithError` is used, and you are retrieving the Mobile SDK values, the timeout value is 500ms. If the operation times out is not successful, an `AdobeError` is returned."

(The sentence above that starts with "When the..." should line up with "Added".  Also the bullets are not displaying correctly; the 4 bullets are sub-bulleted list, so they should be indented.)